### ISSUE TITLE
A question about distance in loss

### ISSUE THREAD

**Issue by HWZX** [beihaifusang/NER_Transfer_Learning-LSTM_with_Rule_Model#2]
I wonder why you use exp() on log_likehood when updating the weight
`same_distr_distance = np.exp(sess.run(log_likelihood, feed_dict={x:batch_x, y:batch_y, sentence_lengths:batch_sentences_len}))
    feeder.update_weights(same_distr_distance, batch_weights, diff_distr_distance)

error_rate = np.sum(same_distr_distance*same_distr_weights) / np.sum(same_distr_weights)`

but not use it when calculating the loss of train_data
`loss = tf.reduce_sum(-log_likelihood*weights, name='loss') # mean error over batchß`

**Comment by beihaifusang** [comment]
The distance is used to update the weight instead of training the model given fixed weights.
